resources/views/macros/base.twig

```diff
@@ -36,8 +36,19 @@
     </a>
 {% endmacro %}
 
-{% macro info(text) %}
-    <span class="help-block">{{ _self.icon('info-circle') }}{{ text }}</span>
+{% macro info(text, raw) %}
+    <span class="help-block">
+        {{ _self.icon('info-circle') }}
+        {%- if raw|default(false) -%}
+            {{ text|raw }}
+        {%- else -%}
+            {{ text }}
+        {%- endif -%}
+    </span>
+{%- endmacro %}
+
+{% macro entry_required(text) %}
+    <h4 class="help-block">{{ _self.icon('exclamation-triangle', 'info') }}{{ text }}</h4>
```

Comment:
So, I shall do the following things?
- change the _entry_required_ macro to only be an icon wrapper.
- the macro to also contain the text shall be defined inside the page in which it is used.
- change the h4 to be a paragraph. (I was using h4 because that's how it was in the old page, too. Still, you are right.)